config/redirects

```diff
@@ -4,3 +4,5 @@ define: versions v4.3 v4.4 master
 
 raw: ${prefix}/ -> ${base}/current/
 raw: ${prefix}/master -> ${base}/upcoming/
+
+raw: ${prefix}/${version}/fundamentals/versioned-api/ -> ${prefix}/${version}/fundamentals/stable-api/
```

Comment:
Comment: I think the `raw` redirect doesn't let you use a [versioned variable
](https://docs.mongodb.com/meta/redirects/#rules)

```suggestion
[*-master]: ${prefix}/${version}/fundamentals/versioned-api/ -> ${prefix}/${version}/fundamentals/stable-api/
```

Reply:
Didn't know that, thanks